tests/lib/azure/id.go

```diff
@@ -0,0 +1,7 @@
+package azure
+
+import "fmt"
+
+func getID(resource string, method string, name string) string {
```

Comment:
I thought `getID` was something different until I saw the implementation. I think this name is misleading also. Maybe something like `newID` or `makeID`.